### PR TITLE
Qt updates

### PR DESCRIPTION
**Pull request by Orphis**

### CODE REVIEW

File: Qt/mainwindow.cpp [hunk 1]
```diff
@@ -25,6 +25,8 @@ MainWindow::MainWindow(QWidget *parent, bool fullscreen) :
 	memoryTexWindow(0),
 	displaylistWindow(0)
 {
+    Q_INIT_RESOURCE(desktop_assets);
```
Comment (unknownbrackets) [inline Qt/mainwindow.cpp]: nit: I recommend using `editorconfig`.  See here:

http://editorconfig.org/

We have a file that specifies the line ending / indentation styles we use.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]
LGTM.

-[Unknown]

**Comment by hrydgard** [comment]
Travis is unhappy - other than that, this is great!

File: Qt/mainwindow.cpp (hunk 1 above)
Comment (Orphis) [inline Qt/mainwindow.cpp]: I suggest we clang format everything actually.

**Comment by Orphis** [comment]
It's a classic "works locally, fails in ci" :(

**Comment by hrydgard** [comment]
:(

**Comment by hrydgard** [comment]
RCC: Error in '/home/travis/build/hrydgard/ppsspp/Qt/desktop_assets.qrc': Cannot find file 'languages/ppsspp_de.qm'

Some kind of local language setting issue?

**Comment by Orphis** [comment]
I don't really know. I'm thinking it's not using the same CWD. Trying to build with Qt 5.2.1 as I speak now.

**Comment by Orphis** [comment]
So building on Mac with Make and Qt 5.2.1 works. I'm running out of ideas currently. Removing the translations to see if that temporarily helps.

**Comment by hrydgard** [comment]
Need a rebase!

**Comment by Orphis** [comment]
I'd like to get Appveyor working first, then I'll rebase this and hope I can get the resources working quickly!

**Comment by unknownbrackets** [comment]
Does the remove translations commit mean that the Qt UI (the menus, not the GL parts of the UI) are no longer localized?

-[Unknown]

**Pull request merged**